sdk/remoterendering/mixedreality-remoterendering/src/authentication/mixedRealityAccountKeyCredential.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  AzureKeyCredential,
+  AccessToken,
+  GetTokenOptions,
+  TokenCredential
+} from "@azure/core-auth";
+
+const maxTimestampMs = 8640000000000000;
+
+/**
+ * Represents an object used for Mixed Reality account key authentication.
+ * @internal
+ */
+export class MixedRealityAccountKeyCredential implements TokenCredential {
+  private readonly accountKey: AzureKeyCredential;
```

Comment:
make this `KeyCredential` instead, then you don't have to worry about pulling in `AzureKeyCredential` and calling a constructor, you can use a simple object for the string case